src/lib/features/client-feature-toggles/client-feature-toggle.controller.ts

```diff
@@ -357,11 +357,11 @@ export default class FeatureController extends Controller {
     }
 
     async calculateMeta(query: IFeatureToggleQuery): Promise<IMeta> {
-        // TODO: We will need to standardize this to be able to implement this a cross languages (Edge in Rust?).
```

Comment:
This probably came from a doubt when we were first implementing this. But I think both Unleash and edge should be resolving this in the same way

Reply:
Not really. Edge inherits the meta information, but does its own etag calculation. But I agree with removing the comment.